<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="selectBox">
         <property name="title">
          <string>Select Serial Port</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="descriptionLabel_2">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="manufacturerLabel_2">
            <property name="text">
             <string>Manufacturer:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="pidLabel_2">
            <property name="text">
             <string>Product ID:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="locationLabel_2">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="serialNumberLabel_2">
            <property name="text">
             <string>Serial number:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="vidLabel_2">
            <property name="text">
             <string>Vendor ID:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="serialPortInfoListBox_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="parametersBox">
         <property name="title">
          <string>Select Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QComboBox" name="dataBitsBox_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="parityLabel_2">
            <property name="text">
             <string>Parity:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="parityBox_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dataBitsLabel_2">
            <property name="text">
             <string>Data bits:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="flowControlLabel_2">
            <property name="text">
             <string>Flow control:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="flowControlBox_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="baudRateLabel_2">
            <property name="text">
             <string>BaudRate:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="stopBitsBox_2"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="baudRateBox_2_2"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="stopBitsLabel_2">
            <property name="text">
             <string>Stop bits:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="additionalOptionsGroupBox_2">
         <property name="title">
          <string>Additional options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="localEchoCheckBox_2">
            <property name="text">
             <string>Local echo</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>96</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="applyButton_2">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
